contracts/MarginBase.sol

```diff
@@ -264,6 +303,9 @@ contract MarginBase is MinimalProxyable {
         // define market via _marketKey
         IFuturesMarket market = futuresMarket(_marketKey);
 
+        if (_abs(_depositSize) > freeMargin())
```

Comment:
No `{ }` ? I realize that works but I'd add them since we use them everywhere else.